scripts/script.sh

```diff
@@ -61,8 +63,23 @@ set -o nounset
 ################################
 # This ensures that everything we expect to be in $PATH, actually is.
 
+case "${OS_DISTRIBUTION}" in
+    "Ubuntu")
+        PKG_MGR="apt"
+        BSDTAR_PKG_NAME="libarchive-tools"
+        ;;
+    "openSUSE Leap" | "SLES")
+        PKG_MGR="zypper"
+        BSDTAR_PKG_NAME="bsdtar"
+        ;;
+    *)
+        echo "ERROR: OS distribution ${OS_DISTRIBUTION} is not supported."
+        exit 1
+esac
+
+
 ${PROG_WHICH:="which"} \
-    ${PROG_APT:="apt"} \
+    ${PKG_MGR} \
```

Comment:
Is this correct here, with no assignment?  Is it needed?

Reply:
Yes, it will still checking for the respective OS's package manager to make sure it exist.